app/models/medical_shift.rb

```diff
@@ -5,16 +5,34 @@ class MedicalShift < ApplicationRecord
 
   monetize :amount
 
-  belongs_to :hospital
   belongs_to :user
 
   scope :by_hospital, MedicalShifts::ByHospitalQuery
   scope :by_month, MedicalShifts::ByMonthQuery
   scope :by_payd, MedicalShifts::ByPaydQuery
 
   validates :workload, presence: true
-  validates :date, presence: true
+  validates :start_date, presence: true
+  validates :start_hour, presence: true
   validates :amount_cents, presence: true
 
   validates :amount_cents, numericality: { greater_than_or_equal_to: 0 }
+
+  before_create :parsed_shift
```

Comment:
Hey, I think you need to check the idea of not having the columns `title` and `shift`. Let me know if you have issues while coding it, I can take a look.

Reply:
It makes sense. Suggestions done! b7eb5765fdd3f21007cb71763777e27290c7bead